app/src/main/java/org/gdg/frisbee/android/special/TaggedEvent.java

```diff
@@ -0,0 +1,97 @@
+package org.gdg.frisbee.android.special;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import org.gdg.frisbee.android.R;
+import org.joda.time.DateTime;
+
+public class TaggedEvent implements Parcelable {
```

Comment:
You are right I will change it

On Tue, Feb 24, 2015, 6:40 PM Friedger Müffke <notifications@github.com>
wrote:

> In app/src/main/java/org/gdg/frisbee/android/special/TaggedEvent.java
> <https://github.com/gdg-x/frisbee/pull/215#discussion_r25267452>:
>
> > @@ -0,0 +1,97 @@
> > +package org.gdg.frisbee.android.special;
> > +
> > +import android.os.Parcel;
> > +import android.os.Parcelable;
> > +
> > +import org.gdg.frisbee.android.R;
> > +import org.joda.time.DateTime;
> > +
> > +public class TaggedEvent implements Parcelable {
>
> It is more EventSeries or TaggedEventSeries
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/215/files#r25267452>.
>